internal/operator/install_plan.go

```diff
@@ -35,17 +35,17 @@ func (c operatorClient) InstallPlanApprove(namespace string) error {
 	err = c.Client.Get(context.Background(), types.NamespacedName{Name: installPlanList.Items[0].ObjectMeta.Name, Namespace: namespace}, &installPlan)
 
 	if err != nil {
-		logger.Errorf("no installPlan found in namespace %s: %s", namespace, err)
+		logger.Errorf("no installPlan found in namespace %s: %w", namespace, err)
 		return err
 	}
 
 	if installPlan.Spec.Approval == operatorv1alpha1.ApprovalManual {
 
 		installPlan.Spec.Approved = true
-		logger.Debugf("InstallPlan %s approved by opcap on namespace %s", installPlan.ObjectMeta.Name, namespace)
+		logger.Debugw("InstallPlan Approved", installPlan.ObjectMeta.Name, "Namespace", namespace)
```

Comment:
Debugw format (msg string, keysAndValues ...interface{}); installPlan.ObjectMeta.Name needs a key, "Namespace" should be "namespace"